src/main/java/BlueTurtle/warnings/PMDWarning.java

```diff
@@ -62,25 +60,25 @@ public boolean equals(Object other) {
 
 		return (filePath.equals(that.filePath) && fileName.equals(that.fileName) && line == that.line
 				&& classification.equals(that.classification) && packageName.equals(that.packageName) && type.equals(that.type) 
```

Comment:
In PMD, there are no default messages, and ruleSet is used to concatenate with ruleName to look up in the gdc, so I will change this back.